.github/workflows/build-and-deploy.yml

```diff
@@ -145,12 +145,12 @@ jobs:
     needs: [deploy_to_production]
     uses: ./.github/workflows/starknet-rs-tests.yml
     secrets:
-      STARKNET_RPC: ${{ secrets.PROD_GOERLI_URL }}/v0_4
+      STARKNET_RPC: ${{ secrets.PROD_GOERLI_URL }}/v0_5
```

Comment:
I used it directly in each job to avoid unnecessary complexity. Setting up a global variable in GitHub Actions requires an initial setup job to define the variable and then passing it to each job. Moreover, we can't use such a variable directly; we'd need to declare it as an env variable in every job.